### ISSUE TITLE
Error when running the list_of_contents.py program for a reporting event containing invalid methodId values

### ISSUE THREAD

**Issue by ASL-rmarshall** [cdisc-org/analysis-results-standard#224]
The following error is raised when the list_of_contents.py program in the utilities/python folder is run for a reporting event that contains invalid `methodId` values:

> TypeError: can only concatenate str (not "int") to str

The error occurs because there are missing parentheses on line 433 of the program.  The error can be resolved by adding the parentheses as indicated below:

```python
                            f"{'  '*(level+1)}Method: {analysis.methodId} (**INVALID METHOD ID**)\n"
                                    ^       ^
```
